appveyor.yml

```diff
@@ -88,6 +88,7 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: conda install --yes --quiet conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
+    - cmd: conda install --yes --quiet conda-build=1.21.5
```

Comment:
We need to remove this before merging the PR. Just wanted to make sure this works.

PS: This is not needed in the feedstock.